draft-ietf-rtcweb-jsep.xml

```diff
@@ -2378,6 +2378,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             for bundled m= sections, as
             described in <xref target="I-D.ietf-mmusic-sdp-bundle-negotiation"/>,
             Section 8.1.
+            Note that if media m= sections are bundled into a
+            data m= section, then the TRANSPORT and IDENTICAL attributes
+            may appear in the data m= section even if they would otherwise
+            only be appropriate for a media m= section (e.g., "a=rtcp-mux").
+            This cannot happen in initial offers because all BUNDLE policies
```

Comment:
This explanation is not obvious to me. Yes, at least one m= section must not have a=bundle-only, but that is a truism. The salient point is that initial offers always try to bundle into a non-data section (regardless of bundle policy), or have no media attribs (e.g. the data-only case). I think it would be good to expound on this a bit more.

Also, BUNDLE->bundle.